.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# Generated by <circle ci pipeline new>; this file *should* be edited manually to add or modify steps.
+#
+# Steps are executed based on the version in version control, and so you *do not* need to upload changes to BuildKite, just
+#  commit them and send a PR to GitHub as you normally would.
+#
+# You may find the example pipeline steps listed here helpful: https://buildkite.com/docs/pipelines/defining-steps#example-pipeline but please
+#  note that the setup is already done, so you should not manually adjust anything through the BuildKite interface.
+#
+agent_queue: &agent_queue "queue=v-3f64d560373e7594-s-m-u-1533940994"
```

Comment:
This queue should probably match the other one, right?  `-s-m-u-` isn't a great agent to use, IIRC.

Reply:
Oops, yeah - I intended to fix this and it slipped my mind. I'm going to build a clean copy of this build agent once I have the changes merged into master and then I'll point it at that